data/routes/projectRoute.js

```diff
@@ -0,0 +1,58 @@
+const knex = require("knex");
+const knexConfig = require("../../knexfile");
+const db = knex(knexConfig.development);
+const express = require("express");
+const router = express.Router();
+const helper = require("../helpers");
+
+router.use(express.json());
```

Comment:
Since all routers will use express.json, it would be better to set the use of this in your server.js file prior to where you routes are invoked.